examples/live_tiffs.py

```diff
@@ -0,0 +1,79 @@
+"""
+Loads and Displays tiffs as they get generated in the specific directory.
+Trying to simulate the live display of data as it gets acquired by microscope. 
+"""
+
+import os
+import sys
+import time
+import numpy as np
+from skimage.io.collection import alphanumeric_key
+from dask import delayed
+import dask.array as da
+from tifffile import imread
+import napari
+from napari.qt import thread_worker
+
+
+with napari.gui_qt():
+    viewer = napari.Viewer(ndisplay=3)
+    # pass a directory to monitor or it will monitor current directory.
+    path = sys.argv[1] if len(sys.argv) > 1 else '.' 
+    path = os.path.abspath(path)
+    end_of_experiment = 'final.log'
+
+    def append(delayed_image):
+        if delayed_image is None:
+            return
+
+        if viewer.layers:
+            layer = viewer.layers[0]
+            image_shape = layer.data.shape[1:]
+            image_dtype = layer.data.dtype
+            dask_image = da.from_delayed(
+                delayed_image,
+                shape=image_shape,
+                dtype=image_dtype,
+            ).reshape((1,) + image_shape)
+            layer.data = da.concatenate(
+                (layer.data, dask_image), axis=0
+            )
+        else:
+            image = delayed_image.compute()
+            dask_image = da.from_delayed(
+                delayed_image,
+                shape=image.shape,
+                dtype=image.dtype,
+            ).reshape((1,) + image.shape)
+            layer = viewer.add_image(dask_image, rendering='attenuated_mip')
+
+        if viewer.dims.point[0] >= layer.data.shape[0] - 2:
+            viewer.dims.set_point(0, layer.data.shape[0] - 1)
+   
+  
+    @thread_worker(connect={'yielded': append})
+    def watch_path(path):
```

Comment:
Are you trying to avoid using a directory watcher like [watchdog](https://github.com/gorakhargosh/watchdog) because you don't want another dependency for this example?

Reply:
Yes, thats the main reason.  The other reason is I tried watchdog it has its own thread pool going on that just doesn't want to play along with Qt thereads (Napari Main Thread). I had QFileSystemWatcher based example as well but @jni suggested to go with as minila as possible.

Reply:
Oh really? That's potentially worrying, I don't remember having that particular problem.

Reply:
@GenevieveBuckley Yes, tbh I didnt give much of time on watchdog as it wasn't very simple to get it working with Napri. I am sure we can get it working just that will be lots of code. In current version we can see the important part of loading the files as the get generated. For sure if we want to make it a more robust tool we can do few more things.